lnwallet/channel.go

```diff
@@ -5283,7 +5278,7 @@ func (lc *LightningChannel) ReceiveRevocation(revMsg *lnwire.RevokeAndAck) (
 
 	remoteHTLCs := lc.channelState.RemoteCommitment.Htlcs
 
-	return fwdPkg, addsToForward, settleFailsToForward, remoteHTLCs, nil
```

Comment:
`addsToForward` and  `settleFailsToForward` can be removed in this commit?  We're still appending to the slices on 5172 and 5186 respectively, but not using them.